src/default_event_handler.ts

```diff
@@ -0,0 +1,38 @@
+import type { EventHandler } from "./types";
+
+export class DefaultEventHandler implements EventHandler {
+  constructor() {
+    this.mapHandlers = {
+      click: () => {},
+      dblclick: () => {},
+      mousemove: () => {},
+      mousedown: () => {},
+      mouseup: () => {},
+      mouseout: () => {},
+      dragstart: () => {},
+    };
+    this.documentHandlers = {
+      click: () => {},
```

Comment:
Since we don't have any uses for this yet an it has confusing overlap with the map one, can we remove till we need it?